### ISSUE TITLE
Selectors don't work - the transformation is always applied on the root SVG node.

### ISSUE THREAD

**Issue by wujekbogdan** [JetBrains/svg-mixer#50]
That's my SVG:

``` html
<svg xmlns="http://www.w3.org/2000/svg">
  <path class="path-1" d="..."/>
  <path class="path-2" d="..."/>
  <path class="path-3" d="..."/>
</svg>
```
That's my CSS:

``` css
.test {
    background: url('my-svg');
    -svg-fill: green 'path.path-2';
}
```

I expected the `fill` property to be set on the second `path` node, but the property is set on the svg element instead.

The output code:

``` html
<svg xmlns="http://www.w3.org/2000/svg" fill="green">
  <path class="path-1" d="..."/>
  <path class="path-2" d="..."/>
  <path class="path-3" d="..."/>
</svg>
```

I also tried `-svg-fill: green svg;` instead of `-svg-fill: green 'path.path-2';`, but it doesn't work either, the second argument seems to be ignored.

**Comment by wujekbogdan** [comment]
It looks like if we use `postcss-move-props-to-bg-image-query` then selectors can't be wrapped in apostrophes. On the other hand they mus be wrapped in apostrophes because the following syntax is invalid and causes SCSS compilation errors:

```
// INVALID SYNTAX:
.test {
    background: url('my-svg');
    -svg-fill: green path.path-2;
}
```

But I found that there's a workaround. We can use ids instead of classes - they don't need to be wrapped in apostrophes.

Example:

``` html
<svg xmlns="http://www.w3.org/2000/svg">
  <path id="path-1" d="..."/>
  <path id="path-2" d="..."/>
  <path id="path-3" d="..."/>
</svg>
```

```
// VALID SYNTAX
.test {
    background: url('my-svg');
    -svg-fill: green #path-2;
}
```

**Comment by kisenka** [comment]
@wujekbogdan fixed in `svg-transform-loader@2.0.7`, please update
